week-7/Homework/mandatory/0-thinking-like-a-programmer/task.md

```diff
@@ -26,12 +26,12 @@ After you've watched these videos I'd like you to answer these questions
 
 ## 1. What do you think the most important quality for a programmer is?
 
-<!-- Write your answer here -->
+problem solving
 
 ## 2. When trying to solve a challenge, what should you do first?
 
-<!-- Write your answer here -->
+Understand the problem
 
 ## 3. What should you do if you get stuck?
 
-<!-- Write your answer here -->
+Go step by step through your solution trying to find where you went wrong.
```

Comment:
I like that! Keeping the mindset of 'baby steps' is very important when programming🙂